recipe/meta.yaml

```diff
@@ -44,6 +44,13 @@ test:
     - conda_smithy.configure_feedstock
   commands:
     - conda smithy --help
+    - git config --global user.name "Testy McTest"
```

Comment:
How does this work if you run `conda build` locally? Does this override your personal git config?

Reply:
Yep :(
actually, I think this can be set to local, going to give it a go

Reply:
so, the git config can be set locally but only if you are in a git repo. When running tests with conda build this does not work. Could also set it by the system but that sounds a bit more precarious than setting it globally. The other option is to create a .gitconfig file, point the test to use that and then clean it up. Not really sure what the best option is for this. I mean, it should be mostly run on ci, so the system setting should be ok, but it's also kind of annoying for building locally. What do you think?

Reply:
You might consider putting this under the `run_test.sh`. That way you can test if there is an existing global git config if there is one we can use that, otherwise we can make one. I think that making a git config for a user who didn't have one already should be ok. When they start using git they *hopefully* fix it and set their own config.

Reply:
cool, I got it in one of these
```
if ! [[ $(git config --get user.name) ]]; then git config --local user.name "Testy McTest"; fi
```

Reply:
Awesome (sorry I missed that :sheep:)